app/views/user_mailer/nsv_bsv_submission_rejected.html.erb

```diff
@@ -12,22 +12,18 @@
         <tr>
           <td class='email-sub-title'>
             <p>Dear <%= @submission.name %>,</p>
-            <p>Thank you for your interest in selling with <%= link_to 'Artsy', 'https://www.artsy.net', target: :_blank %> and for giving us a chance to consider your artwork.</p>
-            <p>Our specialists have reviewed your submission and while it’s a wonderful piece, we unfortunately don’t have demand for this artwork on Artsy at the moment.</p>
-
-            <% if Convection.config.send_new_receipt_email %>
```

Comment:
I don't know really, but it shouldn't be in this template. I thought you would know. I will tag @dariakoko on this.

Reply:
Looks about right. I'd tag @AlicanAkyuz to confirm

Reply:
hello 👋  

this env var was added to control whether or not we should inject this new section to the rejection emails depending on environment. the introducing PR is here: https://github.com/artsy/convection/pull/1269 

I believe it can be removed, if you don't need to send a different email template on staging vs. production.